### ISSUE TITLE
Add Molecular Function & Biological Processes Labels of interest to scatterplot

### ISSUE THREAD

**Issue by HamishGWilliams** [HamishGWilliams/GSEA#2]
Now that we have a basic figure for the DESeq results, we can start to incorporate some of the GSEA/overrepresentation analyses results to the plot.

For the first plot, I want to take the most important/interesting GO's of the DEGs list, and label the points on the plot with these GOI (Gene Ontologies of Interest). This will present an interesting and good visual space to display the results, and overall changes to the physiology and processes occurring to Actinia equina at a species level.

The next plot would attempt to group some of the GSEA results on the plot, to try and show the changes in these genes overall as a group (e.g. if all are being consistently upregulated/downregulated, or a mixture of up and down regulated. This would also be a great visual design for showing the results of the GSEA in line with the DEA. 

By linking each of these steps with visuals such as a figure, with each step iterating on the last step, we can keep a consistent visual representation flow for the results, which should help an audience understand where the results come from and how they are linked together. This should help to drive home the points of the results, and the conclusions that I will draw upon with them.

TL;DR of steps:

1. From the Overrepresentation Analyses, choose the most interesting/related GOs for Oil stress response
2. Annotate the 3rd figure (LFC_BP) with this selection of GOs, putting into context the change in regulation of functions/processes.

3. _I'll add the _ GSEA plot changes to another issue as there is a bit more work that needs to be completed for this prior to adding to the plots
